tools/torch_quant/torch_quant/observer.py

```diff
@@ -116,6 +170,60 @@ def forward(self, x):
             return x
 
 
+class PerChannelMinMaxObserver(Observer):
+    def __init__(self, ch_axis=0, dtype: torch.dtype = torch.qint8,
```

Comment:
Is 0 a reasonable default value for ch_axis? make ch_axis a required argument instead?

Reply:
Yes it is a reasonable default value because the first dim of the weight of most quantizable layers in PyTorch is corresponding to  the output channel dimention. And 0 is also adopted as a default value in the PyTorch's per-channel observer/fake-quantizer.